Web/Edubase.Web.UI/Assets/Scripts/Entry/manage-academy-openings.js

```diff
@@ -244,6 +251,14 @@ const academyOpenings = new Vue({
         self.isProcessing = false;
       });
     },
+    getQueryStringValue: function (queryParameterKey){
+      const queryString = window.location.search;
+      const urlParams = new URLSearchParams(queryString);
+      let value=" "; // This is intentionally set to whitespace so a value is registered to the called Api
+      if (urlParams.has(queryParameterKey)) value = urlParams.get(queryParameterKey);
+
+      return value;
+    },
```

Comment:
Rather you than me on this :)
I think I'll need you to explain it to me

Reply:
:) It may be less typing if I show you or do it over a call - is that okay with you?

Reply:
yes definitely

Reply:
Happy to have the call when ever you are free-ish.